src/main/java/frc/robot/vision/limelights/LimelightEntryValue.java

```diff
@@ -0,0 +1,23 @@
+package frc.robot.vision.limelights;
+
+public enum LimelightEntryValue {
```

Comment:
Look at the comment for the other *EntryValue enum

Reply:
?